CONTRIBUTING.md

```diff
@@ -53,7 +53,22 @@ about Git.
   - Create pull request
 
 The title of the pull request should be prefixed by the component or area that
-the pull request affects. Examples:
+the pull request affects. Valid areas as:
+
+  - *Consensus* for changes to consensus critical code
```

Comment:
We also have repository tags for this level of detail. Also the habit so far has been to enclose things in square brackets.